package.json

```diff
@@ -86,15 +86,15 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.12.5",
-    "@videojs/http-streaming": "3.6.0",
+    "@videojs/http-streaming": "3.7.0",
     "@videojs/vhs-utils": "^4.0.0",
     "@videojs/xhr": "2.6.0",
     "aes-decrypter": "^4.0.1",
     "global": "4.4.0",
     "keycode": "2.2.0",
     "m3u8-parser": "^6.0.0",
     "mpd-parser": "^1.0.1",
-    "mux.js": "^6.2.0",
```

Comment:
Do we need it?

Reply:
Unsure, we need to determine if this is actually necessary, but non-blocking for now